### ISSUE TITLE
[Feature request] Please allow us to remove the "Safely Remove Hardware" option from the context menu.

### ISSUE THREAD

**Issue by visusys** [salvadorbs/Asuite#162]
**Is your feature request related to a problem? Please describe.**
There is no real problem, but it would be a little cleaner to remove this option since I never use it.

**Describe the solution you'd like**
A configuration option to remove "Safely Remove Hardware" from the context menu.

**Describe alternatives you've considered**
None

**Additional context**
![vzoEJgsqMv](https://user-images.githubusercontent.com/33441569/170847799-1c400dfa-f999-4efe-a385-b0dad4e08e3f.png)

Thanks for your consideration! ASuite is the best tray launcher out there.

**Comment by salvadorbs** [comment]
Hello,

the option has already been included in the development versions of asuite.

![image](https://user-images.githubusercontent.com/3731721/173095781-81ae02aa-613c-4943-8111-56c02aac19d9.png)

**Comment by visusys** [comment]
Are there binaries available for the Dev version or do I need to compile myself? Thanks so much!

**Comment by salvadorbs** [comment]
Yes, you can download last win32 build: https://github.com/salvadorbs/Asuite/actions/runs/2476018501

**Comment by visusys** [comment]
Please excuse my stupidity, but I can't find anywhere to download the build on that page. Can you further clarify?

**Comment by salvadorbs** [comment]
Sorry, I meant the win32 build is for windows, but asuite is compiled in 64bit.
